cypress.config.ts

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from 'cypress';
+require('dotenv').config();
+
+type Environment = 'localdev' | 'development';
+
+const environment: Environment =
+  (process.env.NEXT_PUBLIC_ENV as Environment) || 'development';
+
+const baseUrlSites: Record<Environment, string> = {
+  localdev: 'http://localhost:3000',
+  development: 'https://housing-register-development.hackney.gov.uk/',
+};
+
+const baseUrl = baseUrlSites[environment];
+
+export default defineConfig({
+  e2e: {
+    baseUrl,
+    blockHosts: ['*.google-analytics.com', '*.sentry.io'],
```

Comment:
This can probably be removed now that sentry and analytics have been handled elsewhere

Reply:
Cleaned up now.